cmd/rebuilddb2/config.go

```diff
@@ -45,16 +45,17 @@ var (
 
 type config struct {
 	// General application behavior
-	ConfigFile  string `short:"C" long:"configfile" description:"Path to configuration file"`
-	ShowVersion bool   `short:"V" long:"version" description:"Display version information and exit"`
-	TestNet     bool   `long:"testnet" description:"Use the test network (default mainnet)"`
-	SimNet      bool   `long:"simnet" description:"Use the simulation test network (default mainnet)"`
-	DebugLevel  string `short:"d" long:"debuglevel" description:"Logging level {trace, debug, info, warn, error, critical}"`
-	Quiet       bool   `short:"q" long:"quiet" description:"Easy way to set debuglevel to error"`
-	LogDir      string `long:"logdir" description:"Directory to log output"`
-	HTTPProfile bool   `long:"httpprof" short:"p" description:"Start HTTP profiler."`
-	CPUProfile  string `long:"cpuprofile" description:"File for CPU profiling."`
-	MemProfile  string `long:"memprofile" description:"File for mempry profiling."`
+	ConfigFile   string `short:"C" long:"configfile" description:"Path to configuration file"`
+	ShowVersion  bool   `short:"V" long:"version" description:"Display version information and exit"`
+	TestNet      bool   `long:"testnet" description:"Use the test network (default mainnet)"`
+	SimNet       bool   `long:"simnet" description:"Use the simulation test network (default mainnet)"`
+	DebugLevel   string `short:"d" long:"debuglevel" description:"Logging level {trace, debug, info, warn, error, critical}"`
+	Quiet        bool   `short:"q" long:"quiet" description:"Easy way to set debuglevel to error"`
+	LogDir       string `long:"logdir" description:"Directory to log output"`
+	HTTPProfile  bool   `long:"httpprof" short:"p" description:"Start HTTP profiler."`
+	CPUProfile   string `long:"cpuprofile" description:"File for CPU profiling."`
+	MemProfile   string `long:"memprofile" description:"File for mempry profiling."`
+	HidePGConfig bool   `long:"hidepgconfig" description:"If set to true it blocks logging of the PostgreSQL db configuration on system start up. Defaults to false."`
```

Comment:
It's OK to omit mention of the default for boolean switches (on the command line it's run like `--hidepgconfig` with no value).  So you can word it to describe what it means if the switch is provided ("Block the logging...").